src/prompt.ts

```diff
@@ -81,6 +81,9 @@ function normal(options: IPromptConfig, retries = 100): Promise<string> {
         resolve(data || options.default)
       }
     })
-    setTimeout(() => reject(), options.timeout || 10000).unref()
+    setTimeout(() => {
+      process.stdin.end()
+      reject(new Error('Prompt timeout'))
```

Comment:
not having an error object in reject was causing issues later in oclif/config

Reply:
I feel that's something TypeScript should enforce (possibly behind a flag)